### DISCUSSION TITLE
Cad viewer.

### DISCUSSION THREAD

**Discussion by d-sooter** [cognitedata/reveal#4484] [Q&A]
Hi guys i have a question is it possible to use the 3D viewer independently of the CDF?

**Comment by pramodcog** [comment]
Hi,

While we recommend using CDF as the data provider for optimal functionality with `Reveal`, the `Reveal 3D viewer` can still be utilized independently.

You can achieve this by substituting the default data provider with your own. Replace the default data provider with your preferred one at this package: [@reveal/data-providers](https://github.com/cognitedata/reveal/tree/master/viewer/packages/data-providers).

However, please note that some customization work may be required to replace the default data provider with your own.
